scripts/build.sh

```diff
@@ -3,5 +3,43 @@
 cd $(dirname "${BASH_SOURCE[0]}")
 set -euxo pipefail
 
-CGO_ENABLED=0 go build -ldflags "-s -w" -o ./coder-logstream-kube ../
-docker build -t coder-logstream-kube:latest .
+# Set CI to false if not set
+CI=${CI:-false}
+
+
+# Get current architecture
+current=$(go env GOARCH)
+# Arhcitectures to build for
+archs=(amd64 arm64 arm)
```

Comment:
Do we have any automated testing for 32-bit ARM?

Reply:
No I guess. Do we have any testing for others?

Reply:
I guess not! We could probably add an integration test using KinD :-)